lib/__tests__/init.test.ts

```diff
@@ -1,11 +1,11 @@
 import AlgoliaInsights from "../insights";
-
-const credentials = {
-  apiKey: "test",
-  applicationID: "test"
-};
```

Comment:
was unsused.
I will check tslint config to make sure this doesn't occur more: https://github.com/algolia/search-insights.js/issues/62